recipes/lapack/all/conanfile.py

```diff
@@ -0,0 +1,113 @@
+import glob
+import os
+import shutil
+from io import StringIO
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanException
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+
+
+class LapackConan(ConanFile):
+    name = "lapack"
+    license = "BSD-3-Clause"
+    homepage = "https://github.com/Reference-LAPACK/lapack"
+    description = "Fortran subroutines for solving problems in numerical linear algebra"
+    topics = (
+        "lapack"
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False], "visual_studio": [True, False]}
```

Comment:
You can detect Visual Studio with `self.settings.compiler == "Visual Studio"`.

Reply:
The issue here is that the original author added this flag because it isn't possible to compile with Visual Studio (see line 62), but in any case it is possible that Visual Studio consumes it (see line 41). I understand that it's not very clean but removing the option would break everything.

Reply:
See how moltenvk handles this case: https://github.com/conan-io/conan-center-index/blob/6f430257c60ca198e164b5e38333d94cf8062e64/recipes/moltenvk/all/conanfile.py#L111-L118

Reply:
Thanks for the suggestion, I adapted it. It should work but I have no way to test it.